dockertest/output/dockerinfo.py

```diff
@@ -0,0 +1,104 @@
+"""
+Handler for 'docker info'. Runs command, returns output in
+machine-friendly form.
+"""
+
+import collections
+import subprocess
+
+
+class DockerInfo(collections.MutableMapping):
+    """
+    Parser of 'docker info' output
+    """
+
+    def __init__(self, info_string=None, docker_path=None):
+        if info_string is None:
+            if docker_path is None:
+                docker_path = 'docker'
+            info_string = subprocess.check_output(docker_path + ' info',
+                                                  shell=True,
+                                                  close_fds=True)
```

Comment:
"recursive initialization": I don't understand what this means. I don't see recursion anywhere. All `__init__` does is run `docker info` and preserve the values.

"docker_path": all I did was copy from `dockertest/output/dockerversion.py` which has the same docker_version in __init__. I don't see a strong reason for it, but can imagine it being useful for testing. If I remove it, should I remove it from `dockerversion.py` also?